src/components/forms/Checkbox/index.tsx

```diff
@@ -1,14 +1,15 @@
-import Checkbox from '@material-ui/core/Checkbox'
 import React from 'react'
+import { Checkbox } from '@gnosis.pm/safe-react-components'
```

Comment:
@nicosampler, shall we wrap the SRC into a Safe's component or better get rid of this and use `Checkbox` directly where needed?

Reply:
theoretically it should be `our wrapper + material-ui` = `safe-react-components`, so we don't need a wrapper anymore

Reply:
@Agupane, can you do the refactor? Basically remove `src/components/forms/Checkbox/index.tsx`

Reply:
done